jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/deploy/DeployServerlessApplicationDialog.kt

```diff
@@ -32,7 +32,7 @@ import javax.swing.JComponent
 class DeployServerlessApplicationDialog(
     private val project: Project,
     private val templateFile: VirtualFile
-) : LoggingDialogWrapper(project) {
+) : DialogWrapper(project) {
```

Comment:
Do we need telemetry for these dialogs?

Reply:
The telemetry for this was pretty useless (it was DeployServerlessApplicationDialog.OkAction / DeployServerlessApplicationDialog.CancelAction). Aded telemetry in `DeployServerlesssApplicationAction` where it was used.

Will take a look at more uses of this tomorrow